### PR TITLE
Updated doxygen version

### PR DESCRIPTION
**Pull request by nihemstr**
- Updating doxygen version for release.

### CODE REVIEW

**Review by Nox-MSFT — DISMISSED** [review]
Nit: for the version number, do we want 3 parts or 4 parts?

File: src/docs/Doxyfile.in [hunk 1]
```diff
@@ -3,7 +3,7 @@ GENERATE_LATEX = NO
 INPUT = "@DOXYGEN_INPUT_DIR@"
 OUTPUT_DIRECTORY = "@DOXYGEN_OUTPUT_DIR@"
 PROJECT_BRIEF = "Azure IoT Agent that delivers client update from Azure Delivery Update (ADU)."
```
Comment (JeffMill) [inline src/docs/Doxyfile.in:5]: still reads "Azure IoT agent" and "Azure Delivery Update" -- are those correct?

File: src/docs/Doxyfile.in [hunk 2]
```diff
@@ -3,7 +3,7 @@ GENERATE_LATEX = NO
...
-PROJECT_NUMBER = "0.0.0.1"
+PROJECT_NUMBER = "1.0.0.0"
 PROJECT_NAME = "Azure Delivery Update Client (ADUC) IoT Agent"
```
Comment (JeffMill) [inline src/docs/Doxyfile.in:7]: same comment as above.

File: src/docs/Doxyfile.in (hunk 1 above)
Comment (nihemstr) [reply src/docs/Doxyfile.in:5]: Asking esha.

**Review by yuvalg-MSFT — APPROVED** [review]

**Review by yuvalg-MSFT — APPROVED** [review]

**Review by JeffMill — APPROVED** [review]

**Pull request merged**